rust/libnewsboat-ffi/src/utils.rs

```diff
@@ -318,6 +318,17 @@ pub extern "C" fn rs_is_valid_color(input: *const c_char) -> bool {
     })
 }
 
+#[no_mangle]
+pub extern "C" fn rs_get_basename(input: *const c_char) -> *mut c_char {
```

Comment:
Please mark this `unsafe`, because it's possible to provoke undefined behaviour by passing something other than a string as an argument. (I did it for the rest of the functions in this module in #610, but that PR hasn't been merged yet.)